cms-publish-rest/src/test/java/se/simonsoft/cms/publish/rest/PublishResourceTest.java

```diff
@@ -156,8 +162,13 @@ public void publishResourceTest() throws Exception {
 														htmlHelperMock,
 														storageFactoryMock,
 														getVelocityEngine());
-		
-		String releaseForm = resource.getReleaseForm(itemId);
+
+		PublishRelease publishRelease = resource.getRelease(itemId);
```

Comment:
Test coverage failed after moving PublishRelease to 'rest' package.

Reply:
It's very possible. I will fix the tests and add one that tests the JSON output once the implementation is satisfactory.